config/settings.defaults.coffee

```diff
@@ -21,14 +21,12 @@ module.exports =
 
 	redis:
 
-		realtime:	
-			port: process.env["REAL_TIME_REDIS_PORT"] or process.env["REDIS_PORT"] or "6379"	
-			host: process.env["REAL_TIME_REDIS_HOST"] or process.env["REDIS_HOST"] or "localhost"	
-			password: process.env["REAL_TIME_REDIS_PASSWORD"] or process.env["REDIS_PASSWORD"] or ""	
+		pubsub:
+			host: process.env['PUBSUB_REDIS_HOST'] or process.env['REDIS_HOST'] or "localhost"
+			port: process.env['PUBSUB_TIME_REDIS_PORT'] or process.env['REDIS_PORT'] or "6379"
```

Comment:
I think this should be `PUBSUB_REDIS_PORT` instead of `PUBSUB_TIME_REDIS_PORT`.

Reply:
Thanks, fixed.